src/api/v1/types.ts

```diff
@@ -370,3 +370,28 @@ export interface GetTableRowsResponse<Index = TableIndexType, Row = any> {
     ram_payers?: Name[]
     next_key?: Index
 }
+
+export interface GetTableByScopeParams<Index = TableIndexType | string> {
```

Comment:
There is no index type or payer:
```
struct get_table_by_scope_params {
      name        code; // mandatory
      name        table; // optional, act as filter
      string      lower_bound; // lower bound of scope, optional
      string      upper_bound; // upper bound of scope, optional
      uint32_t    limit = 10;
      optional<bool>  reverse;
   };
```

Reply:
lol, then I shouldn't be following the official documentation:

https://developers.eos.io/manuals/eos/latest/nodeos/plugins/chain_api_plugin/api-reference/index#operation/get_table_by_scope